Confirmation/Popups for each RPC

akshaysharmajs opened the pull request.

schnetzlerjoe reviewed (changes requested):
Any text where you do "Confirm to delete address with chain ID - ${request.params.chain_id}" you need to use `Confirm to delete address with chain ID - ${request.params.chain_id}`. Please change all to that.

akshaysharmajs commented:
> Any text where you do "Confirm to delete address with chain ID - ${request.params.chain_id}" you need to use `Confirm to delete address with chain ID - ${request.params.chain_id}`. Please change all to that.

Changed all string injections. This change was failing some tests so I refactored the tests too.
